backend/portal/api/portal_api.py

```diff
@@ -5,7 +5,9 @@
 from urllib.parse import urlparse
 
 from flask import Response, jsonify, make_response
+from packaging import version
```

Comment:
I'm not sure introducing a dependency for something this simple is a good idea. Any reason for which you can't just compare the two strings (or even two parsed numbers)?

Reply:
The original requirement was `>=` but I think now it makes sense to just do a string comparison.